### ISSUE TITLE
Account restricted for violation of terms of service

### ISSUE THREAD

**Issue by kawaMANMI** [codewars/codewars.com#2986]
### Discussed in https://github.com/codewars/codewars.com/discussions/2985

<div type='discussions-op-text'>

<sup>Originally posted by **kawaMANMI** April 12, 2023</sup>
Dear Codewars Team,

 

I have been away from Codewars for over two months. Today, upon checking my account, I was surprised to discover that my Kata marks had dropped from almost 1000 to 428. I haven't engaged with the platform in the last two months, but prior to that, I was regularly practicing and solving katas. Could you please clarify what's happening? I attempted to research the term "Account restricted for violation of terms of service," and it appears to be associated with plagiarizing other users' code. However, I don't think that applies to me because my account was in good standing for an extended period. Additionally, I have already archived the solutions for most of my katas. If there are any issues, please inform me, and I can provide additional information.

 

 

My user name is: KawaManmi

and name is Kawa Manmi

 

https://www.codewars.com/users/KawaManmi/stats

 

 

Kind regards,

Kawa

https://github.com/kawaMANMI

https://www.linkedin.com/in/kawa-manmi-5244ab53/</div>

**Comment by Blind4Basics** [comment]
Hello,

There are a lot of solutions that are either suspicious or just obviously plagiarized, tho. Hence the invalidation + restriction.

> Additionally, I have already archived the solutions for most of my katas. 

I don't see what this has to do with the current matter.

**Comment by Blind4Basics** [comment]
well plagiarized solutions are plagiarized solutions. Nothing to clarify about them. I took another look at your account, in your case I could suppress the restriction. Some solutions could be revalidated, also. But you won't get back all the progress/honor.

Btw: no, solving those kata now would be pointless, about getting back rank/honor, because you already have access to all the solutions. On the other hand, you totally can solve them again to actually learn something from them. Food for mind, better than food for pride. ;)

---

Done. I revalidated what looks valid and unlocked your account, considering you didn't seem to plagiarize extensively. You won't get more solutions/progress/rank back. But if that happens again, the next action might be more drastic.

Cheers
